MAIN - Upgrade dependencies to latest versions

Weetbix opened the pull request:
Bumps the versions to the latest versions ignoring the previous version ranges, and using https://www.npmjs.com/package/npm-check-updates

paschalidi commented:
@Weetbix there is a little error here, maybe have a look. 

```
Running @pika/plugin-build-types...
```
throws 
```
Attempted to generate type definitions, but "typescript" package was not found.
Please install either locally or globally and try again.
```

[here](https://circleci.com/gh/VivyTeam/krypt-web/677?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) you can find the error

Weetbix commented:
Updated

I removed the types bundle as it seems its a bit flakey and its presence of typescript can change depending on the package manager (ie its fine with yarn).

Secondly there is no useful type info generated: https://unpkg.com/@vivy/krypt-web@2.0.0/dist-types/index.d.ts
